.github/workflows/linux.yml

```
@@ -0,0 +1,42 @@
+name: Linux
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version:
+          - "1.0"
```

Let see what is the minimal julia version that passes CI
```suggestion
          - "1.0"
          - "1.1"
          - "1.2"
          - "1.3"
          - "1.4"
```

---

IMHO we don't need the other julia versions, i.e., CI should restrict itself to only
- `1.0` (long-term-support version, currently v1.0.5)
- `1` (latest stable release, currently v1.4.0)
- `nightly` (development branch)

So below I just removed `1.0` because it segfaults for some reason I don't understand (and my attempts to analyze and report it following [this](https://docs.julialang.org/en/v1.0/devdocs/backtraces/#Reporting-and-analyzing-crashes-(segfaults)-1) have failed so far)